apps/gallery/js/thumbnails.js

```diff
@@ -21,7 +21,9 @@
   // Bug 1135256: Having a listener in ThumbnailList prevents them from
   // being garbage collected since ready() holds a strong reference, so
   // we have a singleton listener here.
-  navigator.mozL10n.ready(thumbnails.list.localize.bind(thumbnails.list));
+  thumbnails.list.localize();
+  document.addEventListener('DOMRetranslated',
```

Comment:
Is DOMRetranslated fired on the first translation or only when the locale changes when the app is already running?